docs/resources/repository.md

```diff
@@ -20,6 +20,10 @@ resource "cyral_repository" "minimal_repo" {
         host = "mongodb.cyral.com"
         port = 27017
     }
+
+    mongodb_settings {
```

Comment:
Should we maybe add an example of a replica set repo?

Reply:
We do have. Please check the resource that starts in line `51`. See: https://github.com/cyralinc/terraform-provider-cyral/blame/107228b3fcbac6a59efebf4474e8e521052592a6/docs/resources/repository.md#L51-L77

Reply:
Gotcha, thanks for sharing that link :grin: